lib/kudos.ex

```diff
@@ -103,7 +109,7 @@ defmodule Kudos do
             :umbrella
 
           false ->
-            case is_prod?(dep) do
+            case is_prod?(dep) || include_dev_deps do
```

Comment:
Let's skip calling `is_prod` if we already know we'll include dev dependencies
```suggestion
            case include_dev_deps || is_prod?(dep) do
```